fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -157,6 +170,7 @@ def __init__(
         self._validate_conf()
         # TODO (Min): Type checking for the following
         self._transform = torch.fft.fft if algo is Algo.FFT else _dct_transform  # type: ignore
+        self._inverse_transform = torch.fft.ifft if algo is Algo.FFT else _inverse_dct_transform  # type: ignore
```

Comment:
Would condensing this in a single `if algo is Algo.FFT: ...` be better, or is this fine?

Reply:
this is fine. you could do

a, b = fft, ifft if algo is fft else dct, idct